### ISSUE TITLE
Feature Request: AH Quantity/Recent sales

### ISSUE THREAD

**Issue by Aioko** [EdenServer/eden-web#91]
A place where it tells you how many are currently listed on the AH when searching an item and/or how many have sold in the last X amount of days.

Possible location:
![Stock-RecentSold](https://user-images.githubusercontent.com/66692240/85236475-59e42880-b3ec-11ea-9c67-7e531d0ca3e8.jpg)

**Comment by slassen** [comment]
What do you think about a number in parentheses to the right of the name of the item to note how many are in stock?

**Comment by Aioko** [comment]
Yeah whatever you think looks nicer. Just the information being there would be nice. The sold last X amount of days isn't really needed.  You would have to make it update when you click the "Show Singles" or "Show stacks" as well.
